src/test/java/org/prebid/server/analytics/reporter/greenbids/GreenbidsAnalyticsReporterTest.java

```diff
@@ -59,6 +61,15 @@
 
 public class GreenbidsAnalyticsReporterTest extends VertxTest {
 
+    String EXPECTED_JSON_PATH_BANNER =
+            "src/test/java/org/prebid/server/analytics/reporter/greenbids/greenbids_analytics_payload_banner.json";
+
+    String EXPECTED_JSON_PATH_VIDEO =
+            "src/test/java/org/prebid/server/analytics/reporter/greenbids/greenbids_analytics_payload_video.json";
+
+    String EXPECTED_JSON_PATH_BANNER_FORMAT_NULL =
+            "src/test/java/org/prebid/server/analytics/reporter/greenbids/greenbids_analytics_payload_no_format.json";
+
```

Comment:
instead of JSONs I prefer creating an expected message objects inside the test class